pkgs/tools/misc/oscclip/default.nix

```diff
@@ -0,0 +1,36 @@
+{ fetchFromGitHub
+, python3Packages
+, stdenv
+, writeText
+, lib
+, ...
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "oscclip";
+  version = "0.4.1";
+
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "rumpelsepp";
+    repo = "oscclip";
+    rev = "v${version}";
+    sha256 = "sha256-WQvZn+SWamEqEXPutIZVDZTIczybtHUG9QsN8XxUeg8=";
+  };
+
+  buildInputs = with python3Packages; [ poetry-core ];
```

Comment:
As this is a build-time dependency, I think this should be `nativeBuildInputs`.

I wasn't sure, so I did a quick grep. I only found one case that isn't marked as a native input.

Reply:
Yeah it's a build dependency, fixed it.

Reply:
Other than that, it LGTM, but I'm still pretty new myself.